bpf/lib/maps.h

```diff
@@ -57,15 +57,15 @@ struct bpf_elf_map __section_maps cilium_proxy4 = {
 	.size_key	= sizeof(struct proxy4_tbl_key),
 	.size_value	= sizeof(struct proxy4_tbl_value),
 	.pinning	= PIN_GLOBAL_NS,
-	.max_elem	= 8192,
+	.max_elem	= 512000,
```

Comment:
Is there any reason for this number to be power of 2? If yes, shouldn't this be `524288`? If no, ignore this comment.

Reply:
It doesn't really matter but I agree, it looks prettier. I'll change it.